mrt/meetings/printouts.py

```diff
@@ -1120,6 +1120,7 @@ def _process_export_participants_excel(meeting_id, participant_type):
         data['title'] = p.title.value
         data['first_name'] = p.first_name
         data['last_name'] = p.last_name
+        data['sex'] = getattr(p.language, 'value', '-')
```

Comment:
Looks like a typo here? Probably needs to be `.sex` instead of `.language`?

Reply:
:duck: